examples/05-Q3D/Q2D_Armoured_Cable.py

```diff
@@ -0,0 +1,344 @@
+"""
+Q2D: Cable parameter identification
+---------------------------------------------------
+This example shows how you can use PyAEDT to perform these tasks:
+
+ - Create a Q2D design using the Modeler primitives and importing part of the geometry.
+ - Set up the entire simulation.
+ - Link the solution to a Simplorer design.
+
+ For cable information, see `4 Core Armoured Power Cable <https://www.luxingcable.com/low-voltage-cables/4-core-armoured-power-cable.html>`_
+
+"""
+#################################################################################
+# Perform required imports
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+
+import pyaedt
+import math
+
+#################################################################################
+# Initialize core strand dimensions and positions
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Initialize cable sizing - radii in mm
+
+c_strand_radius = 2.575
+cable_n_cores = 4
+core_n_strands = 6
+core_xlpe_ins_thickness = 0.5
+core_xy_coord = math.ceil(3 * c_strand_radius + 2 * core_xlpe_ins_thickness)
+
+#################################################################################
+# Initialize filling and sheath dimensions
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Initialize radii of further structures incrementally adding thicknesses
+
+filling_radius = 1.4142 * (core_xy_coord + 3 * c_strand_radius + core_xlpe_ins_thickness + 0.5)
+inner_sheath_radius = filling_radius + 0.75
+armour_thickness = 3
+armour_radius = inner_sheath_radius + armour_thickness
+outer_sheath_radius = armour_radius + 2
+
+#################################################################################
+# Initialize armature strand dimensions
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Initialize radii
```

Comment:
```suggestion
# Initialize radii.
```
Is this comment correct as the title refers to armature strand dimension? I thought it might be a copy/paste error.